http/tests/pets.rs

```diff
@@ -0,0 +1,14 @@
+#![cfg(feature = "blocking")]
+
+use gw2lib::{
+	model::game_mechanics::pets::{Pet, PetId},
+	Requester
+};
+
+pub mod setup;
+
+#[test]
+fn all() {
+    let client = setup::setup();
+    assert!(client.all::<Pet, PetId>().is_ok());
```

Comment:
Please use `unwrap` here instead, it will print out the relevant error instead of a simple "failed".

Reply:
Also will add missing test for `ids()`